drivers/clock_control/Kconfig.rpi_pico

```diff
@@ -16,4 +16,10 @@ config RPI_PICO_ROSC_USE_MEASURED_FREQ
 	 Instead of the dts value, use the value measured by
 	 the frequency counter as the rosc frequency.
 
+config RPI_PICO_XOSC_STARTUP_DELAY_MULTIPLIER
```

Comment:
This is a hardware-related parameter.
Could you consider moving to the devicetree definition?
Also, you can add a .yaml file to supporting the xosc.

Reply:
Please check the latest changes. It's now configured via a `raspberrypi,pico-xosc` binding.

Reply:
Gread Job! Thanks!.